tools/ci_check_duplicate_usb_vid_pid.py

```diff
@@ -56,6 +56,7 @@
         "espressif_esp32s3_devkitc_1_n8r8",
     ],
     "0x303A:0x7009": ["espressif_esp32s2_devkitc_1_n4", "espressif_esp32s2_devkitc_1_n4r2"],
+    "0x10C4:0xEA60": ["espressif_esp32s2_devkitc_1_n8r2"],
```

Comment:
```suggestion
    "0x303A:0x7009": ["espressif_esp32s2_devkitc_1_n4", "espressif_esp32s2_devkitc_1_n4r2", "espressif_esp32s2_devkitc_1_n8r2"],
```

Reply:
Thanks, this is exactly the feedback I was looking for! 

When I use the native USB port on the board instead, I show a VID of 303A as expected, but a PID of 0002, should I create a new entry for that PID?

Reply:
no, Circuitpython defines the pid and vid, what you see in windows has nothing to do with that. Use the changes I indicated.

Reply:
Will do, thanks!